src/extensions/default/CodeFolding/foldhelpers/handlebarsFold.js

```diff
@@ -0,0 +1,174 @@
+/**
```

Comment:
I think you have to include the full licence comment here (like in other source files).
Beware the date has to be 2016 - present.
```js
/*
 * Copyright (c) 2016 - present Adobe Systems Incorporated. All rights reserved.
 *
 * Permission is hereby granted, free of charge, to any person obtaining a
 * copy of this software and associated documentation files (the "Software"),
 *....
 *...
```